eng/spotbugs-aggregate-report/pom.xml

```diff
@@ -59,6 +59,7 @@
                 <source>..\..\sdk\core\azure-core\src\samples\java</source>
                 <source>..\..\sdk\core\azure-core-amqp\src\main\java</source>
                 <source>..\..\sdk\core\azure-core-http-netty\src\main\java</source>
+                <source>..\..\sdk\core\azure-core-http-netty\src\samples\java</source>
```

Comment:
Can you also add okHttp in here too please?

Reply:
Looks like this PR adds okHttp #5209.